graphs/kickiip.fbp

```diff
@@ -1,4 +1,5 @@
-# !runtime noflo
+# @runtime noflo-nodejs
+# @name IIPKickTest
```

Comment:
Note: only needed when filename doesn't conform to the desired component name (as is often case with components where we need a separate implementation for noflo-nodejs and noflo-browser)